code/modules/antagonists/changeling/abomination.dm

```diff
@@ -0,0 +1,102 @@
+/*
+	- CHANGELING ABOMINATION -
+
+	Contents:
+		Reagent
+		Monster
+*/
+
+/datum/reagent/shiftium
+	name = "shiftium"
+	description = "A strange stimulant."
```

Comment:
A proper description would be better.

Reply:
I didn't think the name shiftium was gonna stay.